src/test/resources/nonPen.tsv

```diff
@@ -0,0 +1,3 @@
+Gene
+ID1
```

Comment:
question: is it allowed to add an additional column with the gene symbol (that is not used by the tool, but can be informative for the user?)

Reply:
any additional columns are ignored, so you can add everything you like